src/main/java/org/example/daos/JobRoleDao.java

```diff
@@ -0,0 +1,39 @@
+package org.example.daos;
+
+import org.example.models.JobRole;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JobRoleDao {
+    public List<JobRole> getAllJobRoles() throws SQLException {
+        List<JobRole> jobRoles = new ArrayList<>();
+
+        try (Connection connection = DatabaseConnector.getConnection()) {
+            Statement statement = connection.createStatement();
+
+            ResultSet resultSet = statement.executeQuery(
+                    "SELECT jobRoleId, roleName, location, capabilityName, bandName, closingDate FROM job_roles"
+                            + " INNER JOIN capability USING(capabilityId)"
+                            + " INNER JOIN band USING(bandId);");
+
+            while (resultSet.next()) {
```

Comment:
Please exclude what is inside the while to separate private method and surround it with try catch inside the method -> this potentially can throw an exception that we want to handle nicely